### ISSUE TITLE
What is the training set for the posted model?

### ISSUE THREAD

**Issue by Kiris-Wu** [drakelevy/ESPCN-TensorFlow#7]
Could you provide some information about the model like epoch,batch _size,learning rate and training set?
Thanks in advance!
